.github/workflows/pr.yaml

```diff
@@ -180,16 +180,12 @@ jobs:
         build-all --verbose -j$(nproc --ignore=1) -DBUILD_CUGRAPH_MG_TESTS=ON;
         sccache -s;
   telemetry-summarize:
-    runs-on: ubuntu-latest
+    # It is essential that this job run on a self-hosted runner. Otherwise, the job
+    # won't be able to communicate with FluentBit, and you won't be recording anything.
```

Comment:
Please use the same wording as the comment on rmm.

https://github.com/rapidsai/rmm/blob/1af03eb55ce51a376c3df2dc0cdf3c81738b2dd6/.github/workflows/pr.yaml#L146